### ISSUE TITLE
Docs for usage in integration test config

### ISSUE THREAD

**Issue by mkurz** [sbt/sbt-java-formatter#74]
The scalafmt docs explain how to use it when the `it` config is enabled:
https://scalameta.org/scalafmt/docs/installation.html#enable-integrationtest

I think it would be nice to explain the same for `sbt-java-formatter` in its README as well.
@ennru I think following should be correct (?)
```scala
inConfig(IntegrationTest)(JavaFormatterPlugin.toBeScopedSettings)
```
and now you can run `it:javafmt`

**Comment by ennru** [comment]
I added this to #73.

**Comment by mkurz** [comment]
@ennru Nice, thanks!

**Comment by ennru** [comment]
Fixed with #73
